test/CMakeLists.txt

```diff
@@ -18,7 +18,19 @@ set(TOPICS
   Array2m
 )
 
-function(add_performance_test TEST_NAME COMM RMW_IMPLEMENTATION)
+function(add_performance_test TEST_NAME COMM RMW_IMPLEMENTATION SYNC_MODE)
+
+  if(${SYNC_MODE} STREQUAL "sync")
```

Comment:
1. If communication mean is FastRTPS, you can specify the publication mode with the `--disable_async` flag on `performance_test`. In that case you don't need the XML.
2. If communication mean is ROS, and `RMW_IMPLEMENTATION=rmw_fastrtps_cpp` or `RMW_IMPLEMENTATION=rmw_fastrtps_dynamic_cpp`, then you want to load the XML for sync (since that is what it configures). If `SYNC_MODE=async`, then you don't need to load the XML, since async is the default behaviour on the RMW layer.

Reply:
https://github.com/ros2/buildfarm_perf_tests/pull/26/commits/335939cb32228f2f2fd80776559f887e398ea543

Reply:
:+1:  For some reason I cannot resolve the comment